src/__tests__/App.test.tsx

```diff
@@ -32,9 +32,9 @@ it('renders without crashing', async () => {
     </Provider>,
   )
 
-  expect(
-    await screen.findByRole('heading', { name: /dashboard\.label/i }, { timeout: 8000 }),
-  ).toBeInTheDocument()
+  await waitFor(() => {
```

Comment:
This test was failing, so we changed it to match RTL's default timeout behavior, and it now passes.